### ISSUE TITLE
[BUG]: The method "unserialize" regarding to the interface returns "void". So results set always be empty I think.

### ISSUE THREAD

**Issue by djvibegga** [phalcon/cphalcon#14942]
https://github.com/phalcon/cphalcon/blob/d3e94630156a0c4441bf5f72e6ed071941c5908b/phalcon/Mvc/Model/Resultset/Complex.zep#L360

**Comment by Jeckerson** [comment]
https://www.php.net/manual/en/class.serializable.php

```
Serializable {
/* Methods */
abstract public serialize ( void ) : string
abstract public unserialize ( string $serialized ) : void
}
```

**Comment by sergeyklay** [comment]
Fixed in the `4.0.x` branch. Thank you for the bug report.
